src/vm/threads.cpp

```diff
@@ -11747,25 +11749,29 @@ ULONGLONG Thread::QueryThreadProcessorUsage()
 }
 #endif // FEATURE_APPDOMAIN_RESOURCE_MONITORING
 
-CrstStatic Thread::s_initializeYieldProcessorNormalizedCrst;
-int Thread::s_yieldsPerNormalizedYield = 0;
-int Thread::s_optimalMaxNormalizedYieldsPerSpinIteration = 0;
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// YieldProcessorNormalized
+
+// Defaults are for when InitializeYieldProcessorNormalized has not yet been called or when no measurement is done, and are
+// tuned for Skylake processors
+int g_yieldsPerNormalizedYield = 1; // 9 for pre-Skylake
```

Comment:
What does this "9 for pre-Skylake" comment mean?

Reply:
For pre-Skylake processors it would be about 9 yields per normalized yield

Reply:
Ok.  It'd be helpful to expand the comment to highlight that this is an estimate for what the value will be, not that we're somewhere setting this to a hardcoded "9".

Reply:
Done